activerecord/test/cases/adapters/postgresql/legacy_migration_test.rb

```diff
@@ -0,0 +1,49 @@
+require "cases/helper"
+
+class PostgresqlLegacyMigrationTest < ActiveRecord::PostgreSQLTestCase
+  class GenerateTableWithoutBigserial < ActiveRecord::Migration::Compatibility::V5_0
+    def change
+      create_table :legacy_integer_pk do |table|
```

Comment:
Primary key columns cannot be null. Therefore implicit default of primary key should be auto numbered I think. Currently looks like that `Compatibility::V5_0`'s primary key default is not auto numbered.